modules/es-extensions/store/asset/swagger/asset.js

```diff
@@ -166,8 +166,29 @@ asset.manager = function(ctx) {
             return asset.name;
         },
         getVersion: function(asset) {
+            if (!asset.attributes["version"]) {
+                var subPaths = asset.path.split('/');
+                asset.version = subPaths[subPaths.length - 2];
+                asset.attributes["version"] = asset.version;
```

Comment:
we can write this as follows in order to be consistent with the rest of the code base:

asset.attributes.version = asset.version